src/sagemaker/session.py

```diff
@@ -4344,9 +4348,11 @@ def update_endpoint(self, endpoint_name, endpoint_config_name, wait=True):
                 "existing endpoint name".format(endpoint_name)
             )
 
-        self.sagemaker_client.update_endpoint(
+        res = self.sagemaker_client.update_endpoint(
             EndpointName=endpoint_name, EndpointConfigName=endpoint_config_name
         )
+        if res:
```

Comment:
Though UpdateEndpoint will always return response (note that this API is not supported in local mode https://github.com/aws/sagemaker-python-sdk/blob/e75865ff02dcb3f5dc5d940a831843f0da8a76f6/src/sagemaker/local/local_session.py#L383-L393), It's safer to check for None.